README.md

```diff
@@ -866,6 +919,17 @@ The custom options are:
   When disabled, it will forward the user to the cross-device flow in order to attempt to capture a video in another device. If the user is already in a mobile device and it does not support
   MediaRecorder, the unsupported browser error will be shown.
 
+#### auth
+
+This is Authentication step. If you have followed the guidelines specific to including auth, you'll have this step made available. In here, a loading screen is presented to the user to fetch all necessary resources to perform authentication.
```

Comment:
Would a user never not have followed the guidelines specific to including auth? 

In other words: won't users always have this step available (or shouldn't they)?

Reply:
No. This step is only available for clients that do require this. We have different imports for IDV and IDV w/ Auth, so the integrator that does _not_ use auth, won't be hindered by the npm size increase.